application/frontend/controllers/UserController.php

```diff
@@ -81,4 +84,16 @@ public function actionUpdatePassword(string $employeeId)
 
         return $user;
     }
+
+    public function actionExpiring(): array
```

Comment:
You'll probably want to add these `/user/expiring` and `/user/new` endpoints to the `api.raml` file.